src/components/MarketList/index.js

```diff
@@ -236,15 +232,26 @@ class MarketList extends Component {
           <div className="container">
             <div className="row">
               <div className="col-xs-10 col-xs-offset-1 col-sm-12 col-sm-offset-0">
-                <button
-                  type="button"
-                  onClick={this.handleCreateMarket}
-                  className="marketStats__control btn btn-default"
-                  disabled={!this.props.defaultAccount}
-                  title={this.props.defaultAccount ? '' : 'Please connect to an ethereum network to create a market'}
+                <Tooltip
+                  overlay="You need a wallet connected in order to create a market"
+                  trigger={this.props.defaultAccount ? [] : ['hover']}
```

Comment:
Can't you remove the tooltip completely if we don't have an account?

Reply:
@andre-meyer Why? It tells user that he needs to connect a wallet

Reply:
But you set the trigger to []?

Reply:
Oh! I meant if the user does have an account. We don't need the tooltip right?